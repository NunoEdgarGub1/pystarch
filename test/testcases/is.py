x01 = True
x02 = 1 if x01 else None
x03 = (x02 is None)
