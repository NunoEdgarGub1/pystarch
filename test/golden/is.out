x01 Bool
x02 Maybe(Num)
x03 Bool
